task/01-strings-tasks.js

```diff
@@ -55,7 +55,8 @@ function getStringLength(value) {
  *   'Chuck','Norris'  => 'Hello, Chuck Norris!'
  */
 function getStringFromTemplate(firstName, lastName) {
-    throw new Error('Not implemented');
+    let hello = 'Hello';
+    return hello.concat(', ' + firstName + ' ' + lastName + '!');
```

Comment:
Please, use  template literal for solving 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals